tos.md

```diff
@@ -10,11 +10,13 @@ Details are provided below, but in general they:
 
 ## Affiliation
 
-Gradebook is not affiliated with, endorsed, or funded by any educational institution or government entity. As such, FERPA regulations do not apply to any data entered on this site. Additionally, by entering data into Gradebook, users agree that these data are NOT official education records of the user or another party. Gradebook should solely be used as a “What-If” analysis tool and should never be used as an official record of educational information.
+Your Gradebook account is always associated with a school or organization. To remain free, Gradebook has partnerships with some of these organizations. We will generally denote if your account is linked to a partnership, but if you have any concerns, you can always contact us.
```

Comment:
```suggestion
Your Gradebook account may be associated with a school or organization. To remain free, Gradebook has partnerships with some of these organizations. We will generally denote if your account is linked to a partnership, but if you have any concerns, you can always contact us or read our <a href="/privacy-policy">Privacy Policy</a> for more details.
```

Reply:
The point I was trying to get across is:

When we sign a user up, they pick associate themselves with a school or organization, or they're associated with the "generic" organization.

Reply:
I think I will leave it as is. An account is technically always associated with a school, plus linking to the Privacy Policy doesn't seem to make much sense here.